### PR TITLE
README Rework

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
[Badge Suggestions]: https://img.shields.io/badge/Ｒｅｐｏｓｉｔｏｒｙ_Ｓｕｇｇｅｓｔｉｏｎｓ-609926?style=for-the-badge&logoColor=white&logo=GitHub
[Badge Changes]: https://img.shields.io/badge/Ｃｈａｎｇｅｓ-D81939?style=for-the-badge&logoColor=white&logo=GitLFS
[Badge Robot]: https://img.shields.io/badge/Ｉ＇ｍ_Ｈｕｍａｎ-c84f8c?style=for-the-badge&logoColor=white&logo=RuboCop
[Badge Ask]: https://img.shields.io/badge/Ｐｌｅａｓｅ_Ａｓｋ_Ｍｅ-73398D?style=for-the-badge&logoColor=white&logo=AskUbuntu

[Widgets]: https://github.com/MarkedDown/Disable/blob/main/Features/Widgets.md
[Tabs]: https://github.com/MarkedDown/Disable/blob/main/Features/Tabs.md
[Wiki]: https://github.com/MarkedDown/Wiki

<br>
<br>

<div align = center>

[<kbd> <br> Please Check The **Preview** <br> </kbd>][Preview]

</div>
  
<br>
<br>

## ![Badge Suggestions]

  <br>

- ***[Disable unused Widgets][Widgets]***:

  - `Packages`

  - `Environments`

  <br>

- ***[Disable unused Tabs][Tabs]***:

  - `Actions`
  
  <br>
  
- ***[Why not to use GitHub Wiki][Wiki]***

  <br>

<br>
<br>

## ![Badge Changes]

- Added missing license text

  *Fixes GitHub not recognizing / showing the license type*

- General formatting / spacing

- Updated 2 / 4 badges to also use shields io

- Updated badge style

- Added quicklinks

- Referenced links

- Added link to game

<br>
<br>

## ![Badge Ask]

- ***If you have any questions.***

- ***In case you do not agree with some of the changes made in this PR***, <br>
  *please first let me know about them, before disregarding it entirely so*<br>
  *we can possibly find a middle ground and have it not go to waste.*

<br>
<br>

## ![Badge Robot]

<kbd> <br>            No, I'm not a robot.            <br> </kbd>

<kbd> <br>    Yes, you are not the first.    <br> </kbd>

<kbd> <br>     I just made a PR template.     <br> </kbd>

<br>
<br>

[Preview]: https://github.com/ElectronicsArchiver/AdminToolbox

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,112 @@
...
+[Badge Release]: https://img.shields.io/github/v/release/Rnen/AdminToolbox?style=for-the-badge
+[Badge Commit]: https://img.shields.io/github/last-commit/Rnen/AdminToolbox?style=for-the-badge&color=258AAF&logoColor=white&logo=GitHub
+
+
+<!-------------------------------[ Buttons ]----------------------------------->
+
+[Button Issues & Ideas]: https://img.shields.io/badge/Issues_&_Ideas-00A8E1?style=for-the-badge&logoColor=white&logo=AskUbuntu
+[Button Installation]: https://img.shields.io/badge/Installation-569A31?style=for-the-badge&logoColor=white&logo=DocuSign
+[Button Development]: https://img.shields.io/badge/Projects-66459B?style=for-the-badge&logoColor=white&logo=AzureArtifacts
+
```
Comment (Rnen) [inline README.md:112]: Thanks for the PR! I don't get many of those, especially nowadays...
Just got a few suggested changes to your version here
```suggestion
<br>

<div align = center>

[![Badge Release]][Releases]   
[![Badge Downloads]][Releases]   
[![Badge Commit]][Commit]   
[![Badge Watchers]][Watchers]

<br>

# AdminToolbox
*A SMod server plugin for **[SCP: Secret Laboratory]**.*

---

[![Button Installation]][Wiki]   
[![Button Issues & Ideas]][Issues]   
[![Button Development]][Projects]

---

</div>

<br>
<br>

## Features

-   Break, lock & unlock doors <br>
    upon interaction.

-   Teleporting Players

-   Endless Rounds

-   Warp Points

-   Ghost Mode

-   Instantkill

-   Infinite items

-   Jailing players

-   Server statistics


<br>

### Friendly Fire

*For every act of friendly fire, the server* <br>
*console and the log files will take note.* <br><br>
*The Plugin also supports turning off certain* <br>
*types of damage between players*

<br>

### Advanced GodMode

- Turn off damage for players

<sub>*'Advanced' in the sense of `106` with ATGOD* <br>
*cannot be contained, amongst other things.*</sub>

<br>

### Player Statistics

*Keeps track of player statistics*<br>
*like kills, deaths, team-kills & playtime*<br>
*and stores it in dedicated player files*

<br>

### Join Message

*The server window shows a player information* <br>
*for every player that joins the server.*


<br>
<br>

## Licensing

**SCP:SL** and it's development team have been <br>
granted permission under the `CC-BY` license.

<br>


<!----------------------------------------------------------------------------->

[Releases]: https://GitHub.com/Rnen/AdminToolbox/releases/
[Watchers]: https://GitHub.com/Rnen/AdminToolbox/watchers/
[Projects]: https://GitHub.com/Rnen/AdminToolbox/projects/1
[Issues]: https://GitHub.com/Rnen/AdminToolbox/issues
[Commit]: https://GitHub.com/Rnen/AdminToolbox/commit
[Wiki]: https://GitHub.com/Rnen/AdminToolbox/wiki

[SCP: Secret Laboratory]: https://store.steampowered.com/app/700330/SCP_Secret_Laboratory/


<!--------------------------------[ Badges ]----------------------------------->

[Badge Downloads]: https://img.shields.io/github/downloads/Rnen/AdminToolbox/total.svg?style=for-the-badge&color=A9225C&logoColor=white&logo=DocuSign
[Badge Watchers]: https://img.shields.io/github/watchers/Rnen/AdminToolbox.svg?style=for-the-badge&label=Watcher&maxAge=2592000&color=73398D&logoColor=white&logo=Git
[Badge Release]: https://img.shields.io/github/v/release/Rnen/AdminToolbox?style=for-the-badge
[Badge Commit]: https://img.shields.io/github/last-commit/Rnen/AdminToolbox?style=for-the-badge&color=258AAF&logoColor=white&logo=GitHub


<!-------------------------------[ Buttons ]----------------------------------->

[Button Issues & Ideas]: https://img.shields.io/badge/Issues_&_Ideas-00A8E1?style=for-the-badge&logoColor=white&logo=AskUbuntu
[Button Installation]: https://img.shields.io/badge/Installation-569A31?style=for-the-badge&logoColor=white&logo=DocuSign
[Button Development]: https://img.shields.io/badge/Projects-66459B?style=for-the-badge&logoColor=white&logo=AzureArtifacts
```

Comment (ElectronicsArchiver) [reply README.md:112]: > I don't get many of those, especially nowadays...

How unfortunate~

> They are a bit all over the place, so sorry for not submitting suggested changes per line.

Well I literally had no idea where the changes were.

*I copied the changes and had to manually re-add all*
*space and newlines to even see the formatting again.*

It seems you only changed feature descriptions, is this correct?

**Comment by Rnen** [comment]
>[Disable unused Tabs](https://github.com/MarkedDown/Disable/blob/main/Features/Tabs.md)
>* Actions

I was actually planning on adding some actions, so these was enabled quite recently :sweat_smile: 
At least a labeler, and possibly a CI

> [Why not to use GitHub Wiki](https://github.com/MarkedDown/Wiki)

Haven't really been concerned with this. Haven't had the time to document more stuff myself, and so far I've had 0 contributors to any documentation

File: README.md (hunk 1 above)
Comment (Rnen) [reply README.md:112]: > It seems you only changed feature descriptions, is this correct?

Yes, and I moved around a couple of line breaks here and there... :sweat_smile:

Comment (ElectronicsArchiver) [reply README.md:112]: I looked through them and fixed them up.

**Comment by ElectronicsArchiver** [comment]
> I was actually planning on adding some actions, so this was enabled quite recently sweat_smile
At least a labeler, and possibly a CI

I see, well then just ignore it.

Most repos don't use actions but GitHub enables them by default~

> Haven't really been concerned with this. Haven't had the time to document more stuff myself, and so far I've had 0 contributors to any documentation

Well if you move it to a `/Documentation/` folder / branch / repo,
I'd be happy to contribute to it~

File: README.md [hunk 2]
```diff
@@ -1,25 +1,124 @@
...
+*Keeps track of player statistics and* <br>
+*stores them in a dedicated files.*
+
+#### Tracked
+
+- `Team Kills`
+- `Playtime`
+- `Deaths`
+- `Kills`
```
Comment (Rnen) [inline README.md:85]: Just a small change here, to reflect the order the stats are saved in, as well as add a "BanCount" stat i forgot.
```suggestion
- `Kills`
- `Team Kills`
- `Deaths`
- `Playtime` (in minutes)
- `Times Banned`
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,128 @@
...
+*The server window information for* <br>
+*every player that joins the server.*
+
+<br>
+<br>
+
+## Licensing
+
+**SCP:SL** and it's development team have been <br>
+granted permission under the `CC-BY` license.
```
Comment (Rnen) [inline README.md:101]:
```suggestion
**AdminToolbox** and it's development team have been <br>
granted permission by Northwood Studios under the `CC-BY` license<br>
to publish server modifications for the game **SCP:SL**
```

Comment (ElectronicsArchiver) [reply README.md]: Fixed the wording / formatting in the last commit.

**Review by Rnen — APPROVED** [review]
Thanks a lot! :heart: 
The AT project has been needing some love for a while, especially the docs :grin:

**Comment by Rnen** [comment]
Also @ElectronicsArchiver , in case you were interested: https://github.com/Rnen/AdminToolbox/pull/126
No pressure though, I am very thankful for this submission :smile:

**Comment by ElectronicsArchiver** [comment]
Already saw it, turned on all activity for the repo~

Was just about to look through it.

**Comment by ElectronicsArchiver** [comment]
I'll create a draft PR for it.

**Pull request merged**